okd/src/configure.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+
+    cat > /etc/microshift/config.yaml <<EOF
+storage:
+    driver: "none"
+EOF
+
+# disable ovsdb
+rm -rf /usr/lib/systemd/system/ovs-vswitchd.service
+rm -rf /usr/lib/systemd/system/openvswitch.service
```

Comment:
```suggestion
rm -f /usr/lib/systemd/system/ovs-vswitchd.service
rm -f /usr/lib/systemd/system/openvswitch.service
```

Reply:
Instead of removing the service, we could just mask it right?